docs/provisioning/sandboxes.md

```diff
@@ -16,6 +16,14 @@ Deploy a completely open sandbox to allow you to try things out in an eyes-on en
 
 [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Fazure-ml-federated-learning%2Fmain%2Fmlops%2Farm%2Fopen_sandbox_setup.json)
 
+### :exclamation: Important parameters
+
+| Parameter | Description | Values |
+| --- | --- | --- |
+| **compute1SKU** | SKU of the first compute to provision.| ex: `Standard_DS4_v2` |
+| **siloRegions** | List of regions used for the silos. All our samples work with 3 regions. | ex: `["australiaeast", "eastus", "westeurope"]` |
+| **kaggleUsername** and **kaggleKey** | Optional: some of our samples require kaggle credentials to download datasets, this will ensure the credentials get injected in the workspace secret store properly. |
```

Comment:
minor: add here that it can be done at a later stage if need be? (along with link to `add-kaggle-credentials.md`)
I suppose it comes up elsewhere in the doc, but I don't want to assume users will read everything.

Reply:
fixed!